solidity/test/OracleDev_test.ts

```diff
@@ -0,0 +1,92 @@
+import * as h from './support/helpers'
+
+contract('OracleDev', () => {
+  const sourcePath = 'OracleDev.sol'
+  const priceFeed = 'LinkEx.sol'
+  const ethRate = 370160
+  const usdRate = 500000
+  let link: any
+  let ocd: any
+  let usdFeed: any
+  let ethFeed: any
```

Comment:
Loving TypeScript. Wondering if we can set the types of these instead of `any`, but that's probably a problem across other files too and could be a separate PR.